config/core/deployments/webhook.yaml

```diff
@@ -60,8 +60,14 @@ spec:
           value: knative.dev/eventing
         - name: WEBHOOK_NAME
           value: eventing-webhook
-        - name: SINK_BINDING_OPT_OUT_SELECTOR
-          value: "false"
+          # SINK_BINDING_SELECTOR can be either inclusion or exclusion (case insensitive).
+          # If `inclusion` is selected, objects labled as `bindings.knative.dev/include:true`
+          # will be considered by the sinkbinding webhook;
+          # If `exclusion` is selected, objects labled as `bindings.knative.dev/exclude:true`
+          # will NOT be considered by the sinkbinding webhook.
+          # The default is `exclusion`.
+        - name: SINK_BINDING_SELECTOR
```

Comment:
I would expect this to be the selector you want to look for, not the setting to turn on if you want to look for the selector already defined.

Reply:
How about `SINK_BINDING_SELECTION_MODE`?

Reply:
This sounds better, thanks.